File: ADS 2024.1/1%programador/logica_programacao/desafio2.js
//Quantas palvras tem mais de 5 letras

const frase = "Aprenda programação seguindo o 1% programador"

let palavras = frase.split(" ")

let contador = 0;

/*
for(let i = 0; i < palavras.length; i++){
        if(palavras[i].length > 5){
            contador = contador + 1       
        }
    }
*/

for(let palavra of palavras){
    if(palavra.length > 5)
        contador++
}
    print(palavras)
    print(contador)


function print(frase){
    console.log(frase)
}

File: ADS 2024.1/1%programador/logica_programacao/desafio2.py
frase = "Aprenda programação seguindo o 1% programador"

palavras = frase.split(" ")
contador = 0


#for palavra in palavras:
#    if len(palavra) > 5:
#        contador = contador + 1

for j in range(len(palavras)):
    if len(palavras[j]) > 5:
        contador += 1

print(palavras)
print(contador) 